### PR TITLE
test(flink): fix application version flakiness

### PR DESCRIPTION
**Pull request by Serpentiel**
## About this change—what it does

fixes Flink application version flakiness

## Why this way

to make it work :)

### CODE REVIEW

File: internal/sdkprovider/service/flink/flink_application_version.go [hunk 1]
```diff
@@ -186,13 +189,40 @@ func resourceFlinkApplicationVersionCreate(ctx context.Context, d *schema.Resour
...
+				Statement: d.Get("statement").(string),
+				Sources:   sources,
+				Sinks:     sinks,
+			},
+		)
+		if err != nil {
+			return &retry.RetryError{
+				Err: err,
+				Retryable: regexp.MustCompile(
+					"Integration not found: " +
```
Comment (ivan-savciuc) [inline internal/sdkprovider/service/flink/flink_application_version.go:212]: Perhaps we can simplify this like `regexp.MustCompile("Integration not found:")`, whatever goes after service integration is unimportant. 

For example, if someone alters an error message on the back-end, like adding a dot in the end or something else, that would impact us here.

**Review by ivan-savciuc — COMMENTED** [review]
LGTM, a small suggestion for catching the right error

File: internal/sdkprovider/service/flink/flink_application_version.go (hunk 1 above)
Comment (Serpentiel) [reply internal/sdkprovider/service/flink/flink_application_version.go:212]: at least we'll know then if something changed around this API :)

**Review by ivan-savciuc — APPROVED** [review]
LGTM

**Pull request merged**